app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :post, foreign_key: 'post_id'
+  belongs_to :user, foreign_key: 'author_id'
+
+  after_save :comments_update
+
+  private
+
+  def comments_update
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [**OPTIONAL**] **Racha**, great job making the `comments_update` and `like_update` methods private and calling them with the `after_save` call back whenever a new `comment` or `like` is saved to the database. Kudos 🥳 👏 💪. 
So I'd like to recommend that you keep up the great work and do the same for the `update_posts` method in the `Post` model. 👌🤗